.github/workflows/continuous-integration-unit-php.yml

```diff
@@ -24,9 +24,9 @@ jobs:
     continue-on-error: ${{ matrix.experimental == true }}
     strategy:
       matrix:
-        php: [ '7.0', '5.6' ]
-        wp: [ 'latest' ]
-        coverage: [ false ]
+        php: ['7.0', '5.6']
+        wp: ['latest']
+        coverage: [false]
```

Comment:
Wait, how did this sneak into my PR?

Reply:
Hm, when I run `npx prettier --write .github/workflows/continuous-integration-unit-php.yml`, these changes are automatically made.

Maybe @spacedmonkey has a different version of prettier or some different config options, since he committed a file with a different formatting?